draft-duke-httpbis-quic-version-alt-svc.md

```diff
@@ -47,6 +47,10 @@ desirable properties of a QUIC version. This document specifies a new Alt-Svc
 parameter that specifies alternative supported QUIC versions, which
 substantially reduces the chance of this penalty.
 
+Similarly, DNS SVCB and HTTPS Resource Records allow clients to use DNS for
+additional instructions to access a service or resource. This document also
+defines a new SvcParamKey for these Resource Records that specifies the
+specific QUIC versions in use.
```

Comment:
I'm no DNS guru but "clients use DNS for additional instructions" seems a little loose.

Perhaps just a tweak of the existing text like

Similarly, clients can retrieve additional instructions about access to services or resources via DNS SVCB and HTTP Resource Records. This document also defines a new SvcParamKey for these Resource Records, which specifies the specific QUIC versions in use.